### ISSUE TITLE
Always downloading Playwright driver

### ISSUE THREAD

**Issue by SamHennessy** [playwright-community/playwright-go#236]
I noticed that in the current master branch it was always downloading the driver. I found the issue, in PlaywrightDriver.isUpToDateDriver  It's checking the output of PW, for example:
```
/Users/sam/Library/Caches/ms-playwright-go/1.16.0-next-1634703014000/playwright.sh --version
Version 1.16.0-next
```
With the constant:
```
const playwrightCliVersion = "1.16.0-next-1634703014000"
```
Using:
```
	if bytes.Contains(output, []byte(d.Version)) {
		return true, nil
	}
```
The extra `-1634703014000` on the constant is the issue.
I've fixed it locally by adding this to the end of `isUpToDateDriver`:
```
	if strings.HasPrefix(string(output), "Version ") {
		// Get rid of trailing new line
		outputStr := strings.TrimSpace(string(output))
		if len(outputStr) > 8 && strings.Contains(d.Version, outputStr[8:]) {
			return true, nil
		}
	}
```

I'll submit a PR with the change.

If you have any suggestions for changes I'm happy to change.

**Comment by mxschmitt** [comment]
This looks more like a bug upstream caused by the recent monorepo migration. Let me fix it there instead.

**Comment by SamHennessy** [comment]
Sure thing, I'll close the PR. Thanks for your time.

**Comment by mxschmitt** [comment]
This should get fixed with the next roll.

**Comment by AngangGuo** [comment]
I have the same issue with the latest master branch. Every time I run the program, the driver will be downloaded again.
```
2022/01/12 09:06:00 report date is: 2022-01-11
2022/01/12 09:06:06 Downloading driver to C:\Users\caguoa00\AppData\Local\ms-playwright-go\1.16.3-1635814179000
2022/01/12 09:06:16 Downloaded driver successfully
```
go.mod
```
go 1.17

require github.com/mxschmitt/playwright-go v0.1400.1-0.20211124164553-ce64d8145ec8
```

**Comment by mxschmitt** [comment]
This should be fixed in v1.20.0
